fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -614,4 +617,37 @@ private static URI checksumURI( final String checksum ) {
         }
         return null;
     }
+
+    protected void createFixityEvent(final EventBus eventBus, final RdfStream rs,
+                                     final String baseURL, final String agent,
+                                     final String userID) {
+        final FixityEvent fixityEvent = new FixityEvent();
+        fixityEvent.setDate(System.currentTimeMillis());
+        fixityEvent.setType(128);
```

Comment:
@escowles We can post anything we want to the event bus, anywhere we want. Elsewhere in this PR @yulgit1 is posting events to it from `FedoraFixity`. Why do we even need an event type of this kind? Once we're dealing with the internal event bus, we're dealing with _our_ types, not JCR types.